sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/properties/ChatEventId.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.chat.signaling.properties;
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+
```

Comment:
Removing additional header.

```suggestion
package com.azure.android.communication.chat.signaling.properties;

```